test/metric_value_benchmark/jmx_kafka_test.go

```diff
@@ -49,41 +56,72 @@ func (t *JMXKafkaTestRunner) GetAgentRunDuration() time.Duration {
 	return 2 * time.Minute
 }
 
+func downloadFromS3(bucket string, key string, destPath string) error {
+	sess := session.Must(session.NewSession())
+	s3Client := s3.New(sess)
+
+	result, err := s3Client.GetObject(&s3.GetObjectInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(key),
+	})
+	if err != nil {
+		return fmt.Errorf("failed to download from S3: %v", err)
```

Comment:
Can we add more info to the error log
```suggestion
	return fmt.Errorf("failed to download from S3 bucket %s, key %s: %v", bucket, key, err)
```